mkdocs.yml

```diff
@@ -94,6 +94,7 @@ nav:
           - 'Fn lock hotkey': unified/novacustom/fn-lock-hotkey.md
           - 'Battery management': unified/novacustom/battery-management.md
           - 'Fan profiles': unified/novacustom/fan-profiles.md
+          - 'WiFi + Bluetooth radios switch': unified/novacustom/wifi-bt-switch.md
```

Comment:
Adding a separate page for every option doesn't look like a good idea. Why not simply add a section/paragraph in the setup menu documentation? We already added many things that way here for novacustom and contribute to more scattered information each time

Reply:
Agreed, moved to shared setup menu documentation.